config/config.go

```diff
@@ -77,9 +76,8 @@ func Parse(cfg map[string]string) (Config, error) {
 		config.Columns = strings.Split(strings.ToUpper(colsRaw), ",")
 	}
 
-	// Columns in snowflake is uppercase.
```

Comment:
I believe this comment is especially useful now that we called strings.ToUpper below.

Reply:
yep, I returned comment